analysis_options.yaml

```diff
@@ -1,21 +1,19 @@
-# Defines a default set of lint rules enforced for
-# projects at Google. For details and rationale,
-# see https://github.com/dart-lang/pedantic#enabled-lints.
-#include: package:pedantic/analysis_options.yaml
+include: package:flutter_lints/flutter.yaml
 
-# For lint rules and documentation, see http://dart-lang.github.io/linter/lints.
-# Uncomment to specify additional rules.
-# linter:
-#   rules:
-#     - prefer_single_quotes
+linter:
+  rules:
+    avoid_bool_literals_in_conditional_expressions: false
```

Comment:
also enable this the hell did you disable this for, never use literals on conditional expressions, whats the point